packages/frontend/src/components/common/Table/ScrollableTable/TableBody.tsx

```diff
@@ -79,34 +81,89 @@ const TableRow: FC<TableRowProps> = ({
+                        {cell.getIsGrouped() ? (
+                            <>
+                                <button
+                                    {...{
+                                        onClick: (e) => {
+                                            e.stopPropagation();
+                                            e.preventDefault();
+                                            toggleExpander();
+                                        },
+                                        style: {
+                                            cursor: row.getCanExpand()
+                                                ? 'pointer'
+                                                : 'normal',
+                                            border: 'none',
+                                            background: 'none',
+                                            float: 'left',
+                                        },
+                                    }}
+                                >
+                                    <MantineIcon
+                                        icon={
+                                            row.getIsExpanded()
+                                                ? IconChevronDown
+                                                : IconChevronRight
+                                        }
+                                        color={fontColor}
+                                        size="xl"
+                                        style={{
+                                            float: 'left',
+                                            margin: -4,
+                                        }}
+                                    />
+                                    <span
+                                        style={{
+                                            color: fontColor,
+                                            marginLeft: 4,
+                                        }}
+                                    >
+                                        ({row.subRows.length})
+                                    </span>
+                                </button>
+                                {flexRender(
+                                    cell.column.columnDef.cell,
+                                    cell.getContext(),
+                                )}
+                            </>
+                        ) : cell.getIsAggregated() ? (
+                            flexRender(
+                                cell.column.columnDef.aggregatedCell ??
+                                    cell.column.columnDef.cell,
+                                cell.getContext(),
+                            )
+                        ) : cell.getIsPlaceholder() ? null : (
+                            flexRender(
+                                cell.column.columnDef.cell,
+                                cell.getContext(),
+                            )
```

Comment:
Looking at this file, I feel like there's a possibility to clean up some of the styling code and leverage Mantine components more. But we can do that at the end of the stage.

Reply:
this part is very clear. thanks @agha4to !